README.rst

```diff
@@ -937,11 +937,18 @@ is created even if the default values are unchanged.
 Without this, when creating new objects, inline items won't be saved
 unless users change the default values.
 
+``openwisp_utils.admin.CopyableFieldsAdmin``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+An admin class that allows us to change admin fields to read-only fields
+and provides an improved user experience for copying and pasting the field content.
+Useful for auto-generated fields such as UUIDs, secret keys, tokens, etc.
+
 ``openwisp_utils.admin.UUIDAdmin``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-An admin class that provides the UUID of the object as a read-only input field
-(to make it easy and quick to copy/paste).
+An admin subclass of ``CopyableFieldsAdmin`` with ``uuid`` as a copyable field, which is kept
+for backward compatibility and convenience, since many models show ``uuid`` as the only copyable field.
```

Comment:
Please break each line and maintain a similar line length as the rest of the document, applies also to the section above.

In short, keep line length in check when writing docs, code and code comments.

Reply:
noted!